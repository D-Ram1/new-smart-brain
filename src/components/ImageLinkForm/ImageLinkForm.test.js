import React from 'react';
import { shallow } from 'enzyme';
import ImageLinkForm from './ImageLinkForm';

it('renders without crashing', () => {
    expect(shallow(<ImageLinkForm />)).toMatchSnapshot();
})