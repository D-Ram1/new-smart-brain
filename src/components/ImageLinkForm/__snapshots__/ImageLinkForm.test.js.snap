// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <p
    className="f3"
  >
    This Brain will detect faces in .jpg pictures! Give it a link and try it out!
  </p>
  <div
    className="center"
  >
    <div
      className="form center pa4 br3 shadow-5"
    >
      <input
        className="f4 pa2 w-70 center"
        type="tex"
      />
      <button
        className="w-30 grow f4 link ph3 pv2 dib white bg-light-purple"
      >
        Detect
      </button>
    </div>
  </div>
</div>
`;
